app/src/main/java/net/gsantner/opoc/frontend/filebrowser/GsFileBrowserListAdapter.java

```diff
@@ -71,7 +73,7 @@ public class GsFileBrowserListAdapter extends RecyclerView.Adapter<GsFileBrowser
     private final List<File> _adapterDataFiltered; // Filtered list of current folder
     private final Set<File> _currentSelection;
     private File _currentFolder;
-    private final Context _context;
+    private final Activity _activity;
```

Comment:
While yes it's a Activity refererece, it's OK to store it as `Activity _context` variable.